### DISCUSSION TITLE
How to change font type and size in the tree window and the options interface

### DISCUSSION THREAD

**Discussion by urmel1960** [WinEdt-Team/WinEdt-Support#19] [1.1 General Help and Discussion]
Hello everyone

I am trying to change font type and size in the two windows a) tree and b) options interface. These fonts are very small in my setting.

I menawhile have checked all the options in the options interface that I could find - but with no success.

Any advice where to look is highly appreciated
Thanks

**Comment by Alex-Simonic** [comment]
Options Interface:  Projects, Forms, Sounds. Double-click on Forms. You will find a comment:

```
// FONT_MODE
//   0 = Windows Default (e.g. Tahoma)
//   1 = Default Editor Font (e.g. Consolas)
//   2 = Current Document's Font (the same as 1 unless Font Schemes are used)
//   3 = Custom (as specified)
``` 
Specify:

 ```
 // Default Windows Font
  FONT_MODE=3
  // Custom Mode?
  FONT_NAME="Segoe UI"
  FONT_SIZE=10
  FONT_BOLD=0
  FONT_ITALIC=0
```
or something similar... It works.

**Reply by urmel1960** [reply]
Thank you very very much !
